### DISCUSSION TITLE
About Multi-lingual support

### DISCUSSION THREAD

**Discussion by jimmychu0807** [docsifyjs/docsify#1472] [Q&A] [answered]
Hi, is there a documentation page on docsify on how to enable the multi-lingual feature, as the site have on the top right corner? Where is the translated text stored at?

If there is no doc page, it will be great if you could give me a few pointers and I will go figure it myself. Thank you.

**Comment by sy-records** [comment, marked as answer]
You can set up two directories, Use access to different file paths to achieve multilingualism.

For example one for zh-cn and one for en. 

https://mqtt.simps.io/#/en/

https://mqtt.simps.io/#/zh-cn/

code: https://github.com/simps/mqtt/tree/master/docs

**Reply by jimmychu0807** [reply]
thank you @sy-records
